### PR TITLE
Some lint

### PR DESCRIPTION
**Pull request by joaomcteixeira**
Corrects lint in many files (but not all).

### CODE REVIEW

File: src/idpconfgen/components/xmer_probs.py [hunk 1]
```diff
@@ -83,16 +83,17 @@ def __call__(self, parser, namespace, value, option_string=None):
...
-found in the database and the probabilities renormalized."""
+Provide a two column file where the first column is the size of the fragments
+ you wish to allow **in order from smallest to largest** and the second column
+ is the relative probability to select each choice. Both columns must be
+ integers.
+Use "#" to comment lines. If you don't provide any value, the default is
+ fragment sizes: 1 to 5 with relative probabilities (1, 1, 3, 3, 2). You can
+ give any integer to define relativeprobabilities: 10 and 90 will results
+ 10/(10+90) and 90/(10+90) percent. If a Proline residue follows the fragment
+ being built, the additional Prolineangles will also be consider regardless
```
Comment (menoliu) [inline src/idpconfgen/components/xmer_probs.py:94]: space between "Prolineangles"

Comment (joaomcteixeira) [reply src/idpconfgen/components/xmer_probs.py:94]: my fault on that. thanks! nice catch!

**Comment by joaomcteixeira** [comment]
Yes. Also, I don't want to lint your lines of code because that would overwrite your signature there and add my name instead, and that's not fair :wink: So, with `tox -e lint`, you can identify those messages from your additions and correct them. :+1:

if some line overcome 80 chars for a bit you can add `# noqa: E501` in front of the line to deactivate the error message for that line. See https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors

**Comment by menoliu** [comment]
@joaomcteixeira Phew, linting is all done here! Just the `tests`, but we can do that in #225

**Review by menoliu — DISMISSED** [review]

**Comment by joaomcteixeira** [comment]
Heheh. GREAT Work!! That's a good "phew" indeed :wink:
something got broken with the tests,
`E   ImportError: cannot import name 'add_mcsce_subparser' from 'idpconfgen.components.sidechain_packing' (/home/runner/work/IDPConformerGenerator/IDPConformerGenerator/src/idpconfgen/components/sidechain_packing/__init__.py)`

need to have a look, maybe is just a renaming that got a typo.

**Comment by menoliu** [comment]
I remember `add_mcsce_subparser` was not used in `__init__.py` so `lint` told me to remove it

**Comment by joaomcteixeira** [comment]
Yeah, I see what you mean. But the import was there to lift the function from the module where it was defined to that other one. When those cases happen, add the comment to ignore the error `# noqa: X000` where `X000`  is the error code.

**Comment by menoliu** [comment]
@joaomcteixeira thanks for the clarification, nice to see a checkmark :)

**Review by menoliu — APPROVED** [review]

**Pull request merged**